.github/workflows/deploy.yml

```diff
@@ -28,14 +28,19 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
 
+      - uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # v4.0.0
+
       - name: Install and test
         run: |
-          npm install
-          npm run build --if-present
-          npm run test --if-present
+          pnpm install
+          pnpm run build --if-present
+          pnpm run test --if-present
+
+      - name: Bundle
+        run: pnpm run bundle
 
-      - name: Build zip
-        run: npm run bundle
+      - name: Check bundle
```

Comment:
I'm guessing this isn't related to the pnpm check? Seems like we just weren't doing it on deploy before?

Reply:
Yes, correct; an oversight in my last PR to bundle instead of uploading `node_modules`.